tswift.py

```diff
@@ -61,13 +63,22 @@ def __init__(self, title=None, artist=None, url=None):
     def load(self):
         """Load the lyrics from MetroLyrics."""
         page = requests.get(self._url)
+
+        if page.status_code > 200:
+            raise TswiftError("No lyrics available for requested song")
+
         # Forces utf-8 to prevent character mangling
         page.encoding = 'utf-8'
 
         tree = html.fromstring(page.text)
-        lyric_div = tree.get_element_by_id('lyrics-body-text')
-        verses = [c.text_content() for c in lyric_div.find_class('verse')]
-        self._lyrics = '\n\n'.join(verses)
+        try:
+            lyric_div = tree.get_element_by_id('lyrics-body-text')
+            verses = [c.text_content() for c in lyric_div.find_class('verse')]
+        except KeyError:
+            raise TswiftError("No lyrics available for requested song")
+        else:
+            self._lyrics = '\n\n'.join(verses)
+
```

Comment:
I'm able to reproduce the "KeyError" that happens when you load a song which doesn't exist, e.g. `tswift 'Taylor Swift' -s 'Not a song'`. But can you provide an example which triggers a 200 response with no `lyrics-body-text` element?

Reply:
I was only able to to surface that specific error with a 200 response without the `lyrics-body-text` id. The other one is a non-200 response which I got if you spelled the artist name incorrectly or the artist doesn't exist

Reply:
I've been browsing around Metrolyrics but haven't yet found a situation where you would have a 200 response on a valid song lyric link, but you wouldn't have a `lyrics-body-text` element. Was there a particular instance you encountered this problem with?

(In any case, I'm ok with including this change as-is, it seems possible that you could get this error through the `Song(url='...')` constructor, or some other way).

Reply:
Yea, if you attempt to look up a song by an artist that does not exist or is misspelled
https://www.metrolyrics.com/sdfsdf-lyrics-talking-heads.html